src/test/java/io/lettuce/core/cluster/pubsub/RedisClusterPubSubConnectionIntegrationTests.java

```diff
@@ -101,6 +112,32 @@ void testRegularClientPubSubShardChannels() {
         assertThat(channelsOnOtherNode).isEmpty();
     }
 
+    @Test
+    void subscribeToShardChannel() throws Exception {
```

Comment:
The test should be guarded with `@EnabledOnCommand("SSUBSCRIBE)`

Reply:
done